lib/src/tar/export.rs

```diff
@@ -26,6 +26,8 @@ pub const FORMAT_VERSIONS: RangeInclusive<u32> = 0..=1;
 const SYSROOT: &str = "sysroot";
 // This way the default ostree -> sysroot/ostree symlink works.
 const OSTREEDIR: &str = "sysroot/ostree";
+// The ref added in the exported OSTree repo pointing at the base commit.
+const BASEREF: &str = "base";
```

Comment:
Let's bikeshed this a bit...

 - `container-encapsulated-commit`
 - `default`
 - `ostree/container/encapsulated`
 - `ostree/default`

I think I'd at least vote to prefix this with `ostree/` to be clear it's an invented ostree thing.

Reply:
I went with `ostree/encapsulated`. `commit` and `container` felt redundant, and `default` too generic. I like that `encapsulated` highlights that it's purely an encapsulation concept and not something we'd write to disk on import.

Reply:
Cool, SGTM.